src/main/jhipster/src/main/webapp/app/germplasm-manager/germplasm-search.component.ts

```diff
@@ -720,6 +726,13 @@ export class GermplasmSearchComponent implements OnInit {
     }
 
     exportDataAndLabels() {
+        if (this.germplasmList.length === 0) {
+            this.alertService.error('error.custom', {
+                param: 'The filter applied does not return any results to perform this action. Please check'
```

Comment:
any alternative to externalize the error message? @cuenyad

Reply:
done.